src/collective/lazysizes/profiles/plone5/registry.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<registry
+    i18n:domain="plone"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+  <!--
+    BUNDLES - ready to use
+    The compiled bundle files are empty - they are generated/cooked by Plone.
+  -->
+
+  <records prefix="plone.bundles/lazysizes" interface="Products.CMFPlone.interfaces.IBundleRegistry">
+    <value key="last_compilation"></value>
+    <value key="jscompilation">++plone++collective.lazysizes/bundle-lazysizes-compiled.min.js</value>
```

Comment:
I don't understand how this works, you're adding empty files in both cases and that makes no sense to me.

Reply:
I understand this doesn't make too much sense. But this is the reason:
The resource registry expects these bundle files to be existent, but immediately cooks a new bundle file from it's referenced resources if the ``last_compilation`` time isn't set and ``compile`` is set to ``False``.
Better would be to fix the resource registry to not expect a bundle compilation file here and just create one by itself in that case.
The file is empty because it's cooked (concat + minified) anyways. If we would provide a file by ourself, we had to maintain it manually and upgrade it with every upgrade to the lazysizes core file. The cooked bundle is almost the same like the lazysizes core file, except that a requirejs undefine statement is added to the top of the file and a requirejs redefine is added to the end - this is to isolate legacy js from the global requirejs definitions (otherwise we would get a "Mismatched anonymous define" error).
Does it make more sense now?

Reply:
there are a couple of thing we wanted to do and maybe it's time to do it before your fix:

* use Webpack to build the resources of this package (CC @rodfersou)
* update to new lazysizes version which seems to use CommonJS format for plugins

regarding the later, I found this on the RequireJS documentation: http://www.requirejs.org/docs/commonjs.html

we can check if it's better to use the UMD versions of lazysizes or not.